src/main.rs

```diff
@@ -434,28 +434,120 @@ pub fn compute_mosaic(args: Options) {
 }
 
 fn main() {
-    let args = Options::parse();
+    let args = Options::parse(); 
```

Comment:
no change necessary here
```suggestion
    let args = Options::parse();
```